CairoMakie/src/primitives.jl

```diff
@@ -1036,12 +1072,46 @@ function draw_mesh3D(
         valid = Bool[]
     end
 
+
+    # Camera to screen space
+    ts = map(vs) do v
+        clip = projectionview * v
+        @inbounds begin
```

Comment:
Do we not have a function for this?

Reply:
Not really. The CairoMakie projection functions drop the z component which we use for z sorting here. (And that needs to happen after projectionview.) Makie's project function do that as well except for the ones that include transform_func, model and f32convert.

Reply:
Could we maybe improve project instead and use that?